pylintrc

```diff
@@ -38,8 +60,7 @@ load-plugins=
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
 
-disable=R, missing-docstring, fixme, invalid-name, bad-continuation, attribute-defined-outside-init, arguments-differ,
-    attribute-defined-outside-init, star-args, locally-disabled, unused-argument, unused-variable, no-init,
+disable=R, missing-docstring, fixme, invalid-name, arguments-differ, locally-disabled, unused-argument, unused-variable,
```

Comment:
Most additions and changes in this file are just new defaults from `pylint --generate-rcfile`. The only changes I made to the new default rcfile were to these disable= rules and some of the flags we had already changed (e.g., colorized output, no full reports).